python/cudf/cudf/core/column/string.py

```diff
@@ -699,28 +1392,57 @@ def title(self, **kwargs):
         and lowercase the rest.
         This only applies to ASCII characters at this time.
 
+        Equivalent to `str.title()
+        <https://docs.python.org/3/library/stdtypes.html#str.title>`_.
+
+        Returns : Series or Index of object
+
+        See also
+        --------
+        lower
+            Converts all characters to lowercase.
+
+        upper
+            Converts all characters to uppercase.
+
+        capitalize
+            Converts first character to uppercase and remaining to lowercase.
+
+        swapcase
+            Converts uppercase to lowercase and lowercase to uppercase.
+
         Examples
         --------
         >>> import cudf
-        >>> s = cudf.Series(["Hello friend","goodnight moon"])
-        >>> print(s.str.title())
-        ['Hello Friend', 'Goodnight Moon']
-
+        >>> data = ['lower', 'CAPITALS', 'this is a sentence', 'SwApCaSe'])
+        >>> s = cudf.Series(data)
+        >>> s
+        0                 lower
+        1              CAPITALS
+        2    this is a sentence
+        3              SwApCaSe
+        dtype: object
+        >>> s.str.title()
+        0                 Lower
+        1              Capitals
+        2    This Is A Sentence
+        3              Swapcase
+        dtype: object
         """
         return self._return_or_inplace(cpp_title(self._column), **kwargs)
 
-    def slice_from(self, starts=0, stops=0, **kwargs):
+    def slice_from(self, starts, stops, **kwargs):
```

Comment:
So the default parameters were `0`'s because this api previously used to take device pointer as inputs, now that we have shifted to `Series` for both `starts` & `stops` removed the default values and mandated both inputs. Another reason is when these 0's pass down to cython api call they will error.